ios/RCTPagedContacts/RCTPagedContactsModule.m

```diff
@@ -26,46 +26,53 @@ - (instancetype)init
-			 @"givenName": CNContactGivenNameKey,
-			 @"middleName": CNContactMiddleNameKey,
-			 @"familyName": CNContactFamilyNameKey,
-			 @"previousFamilyName": CNContactPreviousFamilyNameKey,
-			 @"nameSuffix": CNContactNameSuffixKey,
-			 @"nickname": CNContactNicknameKey,
-			 @"organizationName": CNContactOrganizationNameKey,
-			 @"departmentName": CNContactDepartmentNameKey,
-			 @"jobTitle": CNContactJobTitleKey,
-			 @"phoneticGivenName": CNContactPhoneticGivenNameKey,
-			 @"phoneticMiddleName": CNContactPhoneticMiddleNameKey,
-			 @"phoneticFamilyName": CNContactPhoneticFamilyNameKey,
-			 @"phoneticOrganizationName": CNContactPhoneticOrganizationNameKey,
-			 @"birthday": CNContactBirthdayKey,
-			 @"nonGregorianBirthday": CNContactNonGregorianBirthdayKey,
-			 @"note": CNContactNoteKey,
-			 @"imageData": CNContactImageDataKey,
-			 @"thumbnailImageData": CNContactThumbnailImageDataKey,
-			 @"phoneNumbers": CNContactPhoneNumbersKey,
-			 @"emailAddresses": CNContactEmailAddressesKey,
-			 @"postalAddresses": CNContactPostalAddressesKey,
-			 @"dates": CNContactDatesKey,
-			 @"urlAddresses": CNContactUrlAddressesKey,
-			 @"socialProfiles": CNContactSocialProfilesKey,
-			 @"instantMessageAddresses": CNContactInstantMessageAddressesKey,
-			 @"relations": CNContactRelationsKey,
-			 };
+	NSMutableDictionary *constants = [@{
+		@"identifier": CNContactIdentifierKey,
+		@"displayName": @"displayName",
+
+		@"namePrefix": CNContactNamePrefixKey,
+		@"givenName": CNContactGivenNameKey,
+		@"middleName": CNContactMiddleNameKey,
+		@"familyName": CNContactFamilyNameKey,
+		@"previousFamilyName": CNContactPreviousFamilyNameKey,
+		@"nameSuffix": CNContactNameSuffixKey,
+		@"nickname": CNContactNicknameKey,
+		@"organizationName": CNContactOrganizationNameKey,
+		@"departmentName": CNContactDepartmentNameKey,
+		@"jobTitle": CNContactJobTitleKey,
+		@"phoneticGivenName": CNContactPhoneticGivenNameKey,
+		@"phoneticMiddleName": CNContactPhoneticMiddleNameKey,
+		@"phoneticFamilyName": CNContactPhoneticFamilyNameKey,
+		@"phoneticOrganizationName": CNContactPhoneticOrganizationNameKey,
+		@"birthday": CNContactBirthdayKey,
+		@"nonGregorianBirthday": CNContactNonGregorianBirthdayKey,
+		@"note": CNContactNoteKey,
+		@"imageData": CNContactImageDataKey,
+		@"thumbnailImageData": CNContactThumbnailImageDataKey,
+		@"phoneNumbers": CNContactPhoneNumbersKey,
+		@"emailAddresses": CNContactEmailAddressesKey,
+		@"postalAddresses": CNContactPostalAddressesKey,
+		@"dates": CNContactDatesKey,
+		@"urlAddresses": CNContactUrlAddressesKey,
+		@"socialProfiles": CNContactSocialProfilesKey,
+		@"instantMessageAddresses": CNContactInstantMessageAddressesKey,
+		@"relations": CNContactRelationsKey,
+	} mutableCopy];
+
+	// CNContactPhoneticOrganizationNameKey is only available in iOS10
+	if (&CNContactPhoneticOrganizationNameKey != nil) {
+    [constants setValue:CNContactPhoneticOrganizationNameKey forKey:@"phoneticOrganizationName"];
```

Comment:
Funny, I was thinking last week this needs to be done to prevent crashes on iOS 9 - and forgot to do. 😆 

One request - please change your coding style to that of the library - tab instead of spaces. If this is too much to ask, I will reject the request and will reimplement the gist on my own. Thanks!